koku/api/report/test/all/openshift/test_query_handler.py

```diff
@@ -289,6 +250,10 @@ def test_multi_exclude_functionality(self):
         """Test that the exclude feature works for all options."""
         exclude_opts = list(OCPAllExcludeSerializer._opfields)
         exclude_opts.remove("source_type")
+        exclude_opts.remove("account")
+        exclude_opts.remove("az")
+        exclude_opts.remove("instance_type")
+        exclude_opts.remove("storage_type")
```

Comment:
A comprehension is more efficient for filtering a list.
```suggestion
        exclude = {"source_type", "account", "az", "instance_type", "storage_type"}
        exclude_opts = [opfield for opfield in OCPAllExcludeSerializer._opfields if opfield not in exclude]
```